src/app/@ansyn/ansyn/modules/menu-items/results/components/results-table/results-table.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Observable } from "rxjs";
+import { IOverlay } from "../../../../overlays/models/overlay.model";
+import { select, Store } from "@ngrx/store";
+import {
+	IOverlaysState, MarkUpClass, selectFilteredOveralys,
+	selectOverlaysArray
+} from "../../../../overlays/reducers/overlays.reducer";
+import { distinctUntilChanged, tap } from "rxjs/operators";
+import { isEqual } from "lodash";
+import {
+	DisplayOverlayFromStoreAction,
+	SetMarkUp,
+} from "../../../../overlays/actions/overlays.actions";
+import { AutoSubscription, AutoSubscriptions } from "auto-subscriptions";
+
+@Component({
+	selector: 'ansyn-results-table',
+	templateUrl: './results-table.component.html',
+	styleUrls: ['./results-table.component.less']
+})
+
+@AutoSubscriptions()
+export class ResultsTableComponent implements OnInit, OnDestroy {
+	overlays = [];
+	selectedOverlayId: string;
+	tableHeaders = [
+		{
+			headerName: 'Date & time',
+			headerData: 'date',
+			isSorted: false
+		},
+		{
+			headerName: 'Sensor',
+			headerData: 'sourceType',
+			isSorted: false
+		},
+		{
+			headerName: 'Type',
+			headerData: 'type',
+			isSorted: false
+		}
+	];
+
+	@AutoSubscription
+	loadOverlays$: Observable<IOverlay[]> = this.store$
+		.pipe(
+			select(selectOverlaysArray),
+			distinctUntilChanged(isEqual),
+			tap((overlays: IOverlay[]) => {
+				this.overlays = overlays;
+			}));
+
```

Comment:
combine `loadOverlays$` and `loadFilteredOverlays$` to one observable,
take `selectFilteredOveralys` with `withLatestFrom`
also i think that distinctUntilChanged(isEqual) is redundant